smolyay/benchmark/beale.py

```diff
@@ -30,3 +30,11 @@ class beale(_beale):
     def domain(self):
         return ([[-7.0000000008, 11.69999999928],
                           [-9.5000000002,9.44999999982]])
+    
+    @property
+    def global_minimum(self):
+        return 0
+    
+    @property
+    def global_minimum_location(self):
+        return [2.9999999992, 0.4999999998]
```

Comment:
Is this how this came from Dr. K? I would assume this is actually (3, 0.5).

Reply:
I was copied straight from the text file. This problem is easy to solve by hand, and I've confirmed that it is (3, 0.5) and I can go ahead and update it.